### PR TITLE
Added nullability to service declaration in objc

### PR DESCRIPTION
**Pull request by BenchR267**
#6162

### CODE REVIEW

**Comment by BenchR267** [comment]
I think your Jenkins has some issues.. 😆

**Comment by jcanizales** [comment]
This is OK to test.

**Comment by jcanizales** [comment]
Travis is failing with:

```
In file included from /Users/travis/build/grpc/grpc/src/objective-c/tests/RemoteTestClient/Test.pbrpc.m:1:
/Users/travis/build/grpc/grpc/src/objective-c/tests/RemoteTestClient/Test.pbrpc.h:10:1: error: cannot #include files inside '#pragma clang assume_nonnull'
#import "Empty.pbobjc.h"
^
/Users/travis/build/grpc/grpc/src/objective-c/tests/RemoteTestClient/Test.pbrpc.h:7:1: note: #pragma entered here
NS_ASSUME_NONNULL_BEGIN
^
```

**Comment by BenchR267** [comment]
Of course, sorry.. I'll fix that as soon as possible :)

**Comment by jcanizales** [comment]
Also, thanks a lot for the PR!

**Comment by BenchR267** [comment]
@jcanizales test is green now :)

File: src/compiler/objective_c_plugin.cc [hunk 1]
```diff
@@ -82,10 +81,12 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
         declarations += grpc_objective_c_generator::GetHeader(service);
       }
 
+      ::grpc::string nonNullBegin = "\nNS_ASSUME_NONNULL_BEGIN\n\n";
```
Comment (jcanizales) [inline src/compiler/objective_c_plugin.cc]: Small nit that I just realized: make these two `static const ::grpc::string kNonNullBegin`?

Comment (BenchR267) [inline src/compiler/objective_c_plugin.cc]: Sorry for the long response time, I was very busy :) Very good proposal, I am not that much into C++ so I am not familiar with all coding styles :) I've changed it in 1843ccb

**Comment by jcanizales** [comment]
The failed test is unrelated, and already tracked as issue #5622, so I'm merging :)

**Pull request merged**